pkg/cli/admin/groups/sync/ad.go

```diff
@@ -4,7 +4,7 @@ import (
 	legacyconfigv1 "github.com/openshift/api/legacyconfig/v1"
 	"github.com/openshift/library-go/pkg/security/ldapclient"
 	ldapquery "github.com/openshift/library-go/pkg/security/ldapquery"
-	"github.com/openshift/oc/pkg/helpers/groupsync"
+	syncgroups "github.com/openshift/oc/pkg/helpers/groupsync"
```

Comment:
How did it work before? I see that `syncgroups` is referenced in the file, but it was not defined before.

Reply:
I have no idea, this was an automatic change

Reply:
witchcraft